### PR TITLE
feat: add a little logo

### PR DESCRIPTION
**Pull request by henryiii**
Adding a small logo.

Similar changes should be done to place a nice error message / traceback into this area - the only difference will be it might need to be scrollable.

### CODE REVIEW

File: src/uproot_browser/plot_view.py [hunk 1]
```diff
@@ -63,14 +64,33 @@ async def update(self) -> None:
         self.refresh()
 
     def render(self) -> rich.console.RenderableType:
-        if self.plot is None or self.plot is EMPTY:
```
Comment (henryiii) [inline src/uproot_browser/plot_view.py:66]: Maybe the best way to handle errors would be to put a try/except block, then put the exception object in as `self.plot`. Then if self.plot isinstance Exception, that gets displayed (probably a rich method somewhere that renders exceptions, should be in a scroll view). You could also set it to a Text string, and if it's Text, that's a "known error" and that gets displayed (like "not a known plotable object" or something, maybe eventually some details like "Show" in ROOT would show.

Comment (henryiii) [reply src/uproot_browser/plot_view.py:66]: Dynamically adding the scroll view might be a problem, if a scroll view is not a rich.console.RenderableType. That can be a second step, or maybe we can manage to classify all the likely errors.

**Comment by henryiii** [comment]
@amangoel185 look okay? Don't want to clash with what you are working on. :)

**Comment by amangoel185** [comment]
Looks good to me! I checked out the new branch from here itself

**Comment by amangoel185** [comment]
In reference to #12

**Pull request merged**